api_tests/base/test_sloan_study.py

```diff
@@ -242,3 +242,16 @@ def test_user_get_cookie_when_flag_is_everyone(self, app, user, preprint):
     def test_get_domain(self, url, expected_domain):
         actual_domain = SloanOverrideWaffleMiddleware.get_domain(url)
         assert actual_domain == expected_domain
+
+    @pytest.mark.enable_quickfiles_creation
+    def test_user_override_cookie(self, app, user, preprint):
+        user.add_system_tag(SLOAN_COI)
+        cookies = {
+            SLOAN_COI_DISPLAY: 'False',
+        }
+
+        resp = app.get('/v2/', auth=user.auth, cookies=cookies)
+
+        cookies = resp.headers.getall('Set-Cookie')
+
+        assert f' dwf_{SLOAN_COI_DISPLAY}=True; Domain=.osf.io; Path=/; samesite=None; Secure' in cookies
```

Comment:
could you also assert that the flag is included in `active_flags` in the response? since that's what FE actually uses, it's arguably more important than the cookie

also add another test (or parametrize this one) to flip true/false -- the `no_` tag should override a `True` cookie